frontend/src/components/Service/ServiceHeatmap.vue

```diff
@@ -82,18 +82,16 @@
       },
       methods: {
           async chartHeatmap() {
-              let start = new Date(new Date().getUTCFullYear(), new Date().getUTCMonth()-2, 1);
+              let start = new Date(Date.UTC(new Date().getUTCFullYear(), new Date().getUTCMonth()-2, 1));
               let monthData = [];
               let monthNum = start.getUTCMonth()
 
               for (let i=1; i<=3; i++) {
-                  let end = this.lastDayOfMonth(monthNum)
-
-                  window.console.log("getting: ",start, end)
+                  let end = this.lastDayOfMonth(start)
 
                   const inputdata = await this.heatmapData(start, end)
                   monthData.push(inputdata)
-                  start = new Date(start.getUTCFullYear(), start.getUTCMonth()+1, 1);
+                  start = new Date(Date.UTC(start.getUTCFullYear(), start.getUTCMonth()+1, 1));
```

Comment:
As the get... methods are already called earlier we can also use variables for these instead of recalculating the values.

Reply:
I thought of using another approach, but I preferred to stay as close as possible to the current code.
The code related to date are always error prone and hard to get right.